rdrf/rdrf/static/css/rdrf.css

```diff
@@ -109,6 +110,18 @@ body {
   }
 }
 
+@media (max-width: 767px) {
+  .staging-env-sm-container {
+    padding-top: 30px;
+  }
+  .staging-env-sm-banner {
+    top: 130px !important;
```

Comment:
I think it would be better to try to cascade the css the ordinary way than to use `!important`